protocol/src/main/java/org/fido/iot/protocol/CryptoService.java

```diff
@@ -441,8 +450,8 @@ public int getCompatibleHmacType(PublicKey key) {
    * @return signature
    */
   public Composite getSigInfoB(Composite sigInfoA) {
-    if (null != sigInfoA && sigInfoA.size() > 0
-            && Arrays.asList(Const.SG_EPIDv10, Const.SG_EPIDv11, Const.SG_EPIDv20)
+    if (null != sigInfoA
```

Comment:
Could you please help me understand why `&& sigInfoA.size() > 0` was removed? 
This condition was added for non-EPID device value of eA. Without this Component Sample Device runs into IndexOutOfBoundException.

Reply:
removed by mistake - will add it back.

Reply:
yes